installer/sakura-common.iss

```diff
@@ -48,70 +48,164 @@ MinVersion=6.1
 [Languages]
 Name: "ja"; MessagesFile: "compiler:Languages\Japanese.isl"
 Name: "en"; MessagesFile: "compiler:Default.isl"; InfoBeforeFile: "instmaterials\info_us.txt"
+Name: "zh_hans"; MessagesFile: "compiler:Languages\ChineseSimplified.isl"; InfoBeforeFile: "instmaterials\info_zh_hans.txt"
+Name: "zh_hant"; MessagesFile: "compiler:Languages\ChineseTraditional.isl"; InfoBeforeFile: "instmaterials\info_zh_hant.txt"
```

Comment:
@EldersJavas 

Please change these lines as specified below.

```
Name: "zh_hans"; MessagesFile: "Languages\ChineseSimplified.isl"; InfoBeforeFile: "instmaterials\info_zh_hans.txt"
Name: "zh_hant"; MessagesFile: "Languages\ChineseTraditional.isl"; InfoBeforeFile: "instmaterials\info_zh_hant.txt"
```

You need to add `ChineseSimplified.isl` file and `ChineseTraditional.isl` file into your git repository's `installer/Languages` folder because they don't exist inside `C:\Program Files (x86)\Inno Setup 6\Languages` folder (as a workaround).

Additionally, you also need to add `info_zh_hans.txt` file and `info_zh_hant.txt` file into your git repository. I think you simply forgot to do so.

Reply:
Sorry for not replying to you for so long.
I have uploaded the file, please check.

Reply:
@EldersJavas  Thank you for uploading the files. 

However, you would still need to rectify `installer/sakura-common.iss` file. Please look up [the comment](https://github.com/sakura-editor/sakura/pull/1469/files#r538538926) carefully. I intentionally removed `compiler:` from `MessageFile:` value. If there's a `compiler:` keyword, the installer would search the specified file from InnoSetup  installed directory, and otherwise the file inside the Source Directory will be archived in the installer and will be used. I guess I should have explained this behaviour as it is not very straight forward to understand.

Here's an excerpt from [Inno Setup Help](https://jrsoftware.org/ishelp/).

> #### MessagesFile  (Required)
> Specifies the name(s) of the .isl file(s) to read the default messages from. The file(s) must be located in your installation's source directory when running the compiler, unless a fully qualified pathname is specified or the pathname is prefixed by `"compiler:"`, in which case it looks for the file in the compiler directory.

Reply:
https://github.com/sakura-editor/sakura/pull/1469/files#r538538926 suggested below.

```suggestion
Name: "zh_hant"; MessagesFile: "Languages\ChineseTraditional.isl"; InfoBeforeFile: "instmaterials\info_zh_hant.txt"
```

this should also be applied to non-traditional version.